src/demo/containers/playground/UpsetPlotDemo.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import UpsetPlot from 'lib/containers/UpsetPlot'
+
+export const UpsetPlotDemo = () => {
+  return (
+    <UpsetPlot
+      entityId='syn20821313'
```

Comment:
I'm trying to revert course on the decision to use the entityId everywhere, if we could change this to grab the entityId from the sql instead.

Reply:
parse entity id